DOC-2171: fix documentation entry for TINY-9975 (x2), TINY-9998 in the 6.7 Release Notes. (#2897)
tinymce/tinymce-docs: modules/ROOT/pages/6.7-release-notes.adoc

==== Applying checklist on a list with nested lists turned only the outer list into a checklist
// TINY-9998

> In previous versions of {productname}, an issue was identified that affected the functionality of the xref:checklist.adoc[Checklist] Plugin when user's would select a checklist that contains a nested checklist and attempted to convert it into a bulleted list or a numbered list.
> 
> As a consequence, the outer checklist would be converted into the desired list while the nested checklists retained the previous formatting state.
> 
> {productname} 6.7 addressed this issue, by ensuring that the whole checklist is now converted into the desired list selected by the user which includes nested checklist.

Reviewer: ```suggestion
Previously, when a xref:checklist.adoc[Checklist] that contained a nested checklist was selected and converted to a bulleted or numbered list, only the outer checklist was converted: the nested checklist remained a checklist.

**Checklist** 2.0.6 corrects this. With this update, nested checklists become nested bulleted lists, or nested numbered lists, when selected and converted, as expected.

For information on the **Checklist** plugin, see: xref:checklist.adoc[Checklist].
```
Suggested edit.

**PowerPaste** 6.2.1 includes the following bug-fixes

==== Stopped pasting comments from Microsoft Word documents.
// TINY-9975
Previously, when a copied selection from a Microsoft Word document included comments, these comments were, added to the {productname} document as footnotes when the selection was pasted in to place.

> **PowerPaste** 6.2.1 addresses this. Comments are still included in copied selections from Microsoft Word. With this upate, however, they are ignored by the plugin and, consequently, not added to {productname} documents when pasted.

Reviewer 2: ```suggestion
**PowerPaste** 6.2.1 addresses this. Comments are still included in copied selections from Microsoft Word. With this update, however, they are ignored by the plugin and, consequently, not added to {productname} documents when pasted.
```

> ==== Stopped pasting comments from word documents.

Reviewer: ```suggestion
==== Stopped pasting comments from Microsoft Word documents.
```
Copy edit: correct nomenclature for referencing Microsoft Word.

==== Stopped pasting comments from word documents.
// TINY-9975

> Previously, pasting content from a word document that contained comments were copied alongside the document's content, resulting in the unintended inclusion of comments when pasting it into a {productname} document.
> 
> {productname} 6.7, introduced a solution for xref:introduction-to-powerpaste.adoc[PowerPaste] that now excludes comments when pasting data into a {productname} document.
> 
> As a result of this fix, users can now paste content from Word documents without the inconvenience of comments being inadvertently included in the paste operation.
> 
> For information on the **PowerPaste** plugin, see: xref:introduction-to-powerpaste.adoc[PowerPaste].

Reviewer: ```suggestion
Previously, when a copied selection from a Microsoft Word document included comments, these comments were, added to the {productname} document as footnotes when the selection was pasted in to place.

**PowerPaste** 6.2.1 addresses this. Comments are still included in copied selections from Microsoft Word. With this upate, however, they are ignored by the plugin and, consequently, not added to {productname} documents when pasted.
```
Suggested edit.

> ==== Removed translations for an error which no longer occurs.
> // TINY-10045

Reviewer: ```suggestion
==== Removed error condition, error message, and error message translations for an error which no longer occurs.
// TINY-10045

Previously, when **PowerPaste** was running with older versions of Safari as the host browser, an error condition presented, along with an associated error message, when direct image pasting was attempted.

This error condition no longer presents and, for this release, the error catching logic, associated error message, and error message translations, have all been removed.

For information on the **PowerPaste** plugin, see: xref:introduction-to-powerpaste.adoc[PowerPaste].
```
For simplicity’s and completeness’s sake, here’s a write-up of the last PowerPaste Release Notes entry. (Note, I moved the _For information…_ line to this — the last — entry in this section.

=== Applying an ordered or unordered list to a selected checklist incorrectly turned the list into paragraphs
// TINY-9975

> In previous versions of {productname}, an issue was identified that affected the functionality of the xref:checklist.adoc[Checklist] Plugin which would incorrectly convert a list into a paragraph when users attempted to apply an `ordered` or `unordered` list format to a selected checklist, particularly when they selected a bulleted list that included a sublist.
> 
> {productname} 6.7 addresses this issue so that when a user selects a bulleted list that contains a sublist and tries to convert it into a checklist, the entire list correctly transforms into a checklist as intended.
> 
> As a result of this fix, lists will now longer convert into paragraphs when attempting to apply checklist formatting to them.
> 
> For information on the **Checklist** plugin, see: xref:checklist.adoc[Checklist].

Reviewer: ```suggestion
Previously, when a xref:checklist.adoc[Checklist], particularly a checklist that contained a nested checklist, was selected and converted to a bulleted or numbered list, the selection was, incorrectly, converted into plain paragraphs.

{productname} 6.7 address this issue. With this update, nested checklists become nested bulleted lists, or nested numbered lists, when selected and converted, as expected.
```
Suggested edit (note, I moved the _For information…_ line to the Checklist write-up).

=== Applying an ordered or unordered list to a selected checklist incorrectly turned the list into paragraphs
// TINY-9975

> Previously, when a xref:checklist.adoc[Checklist], particularly a checklist that contained a nested checklist, was selected and converted to a bulleted list  the selection was, incorrectly, converted into plain paragraphs.

Reviewer: ```suggestion
Previously, when a xref:checklist.adoc[Checklist], particularly a checklist that contained a nested checklist, was selected and converted to a bulleted list the selection was, incorrectly, converted into plain paragraphs.
```

> Previously, when a xref:checklist.adoc[Checklist], particularly a checklist that contained a nested checklist, was selected and converted to a bulleted or numbered list, the selection was, incorrectly, converted into plain paragraphs.

Reviewer 2: Converting a checklist to a numbered list worked fine. The bug only happens when trying to convert a checklist to a bulleted list
```suggestion
Previously, when a xref:checklist.adoc[Checklist], particularly a checklist that contained a nested checklist, was selected and converted to a bulleted list  the selection was, incorrectly, converted into plain paragraphs.
```

> {productname} 6.7 address this issue. With this update, nested checklists become nested bulleted lists, or nested numbered lists, when selected and converted, as expected.

Reviewer: ```suggestion
{productname} 6.7 address this issue. With this update, nested checklists become nested bulleted lists when selected and converted, as expected.
```
